src/lib/bcmp/bcmp_time.cpp

```diff
@@ -2,54 +2,89 @@
 #include "app_util.h"
 #include "bcmp.h"
 #include "bm_util.h"
-#include "device_info.h"
 #include "stm32_rtc.h"
```

Comment:
Is there still a dependency on this?

Reply:
Yeah I put some to-do's to abstract the RTC code away. That should probably be in bm_common right? something like a `bm_rtc.h` or `bm_time.h` file probably?

Reply:
That sounds good! `bm_time` fits more I feel like.